### PR TITLE
chore: Update spartacus and docs repository urls

### PR DESCRIPTION
**Pull request by marlass**
Additionally added repository and homepage
attributes to all published packages to have more
complete information on npm page.

@hackergil The last place I see reference to old repository name is the sonarcloud configuration. Do you think we can safely update that as well?

@gladius-mtl Could you laso check this PR? I'm just not sure if all references should be updated like I did.

@gladius-mtl What do you think about adding README to CDS and core library on npm? Currently those 2 npm packages looks empty: https://www.npmjs.com/package/@spartacus/cds
https://www.npmjs.com/package/@spartacus/core

### CODE REVIEW

File: docs/migration/2_0.md [hunk 1]
```diff
@@ -348,7 +348,7 @@ The HTML and companied CSS is refactored as well. A clean DOM exists of only anc
 
 Using anchor links is the preferred action for pagination links, but action links (using `click` events) are still supported and used in various areas in Spartacus. The product listing page however is using anchor links.
 
-The component is fully [documented](https://github.com/SAP/cloud-commerce-spartacus-storefront-docs/blob/v2-develop/_pages/dev/components/shared-components/pagination.md).
+The component is fully [documented](https://github.com/SAP/spartacus-docs/blob/v2-develop/_pages/dev/components/shared-components/pagination.md).
```
Comment (gladius-mtl) [inline docs/migration/2_0.md:352]: This doc has now been released, so we could update the URL to point to the page on our doc site: https://sap.github.io/spartacus-docs/pagination/

**Review by gladius-mtl — DISMISSED** [review]

**Comment by hackergil** [comment]
We need to test sonar to be sure it won't break and that it can be renamed.

File: docs/migration/2_0.md (hunk 1 above)
Comment (marlass) [reply docs/migration/2_0.md:352]: Updated

**Comment by marlass** [comment]
> We need to test sonar to be sure it won't break and that it can be renamed.

Seems like we need some additional work to rename reference in sonarcoud. Leaving that for another PR. Travis was failing

**Review by gladius-mtl — APPROVED** [review]

**Pull request merged**